net/remote/remote-node.sh

```diff
@@ -80,7 +80,6 @@ local|tar)
       args=(
         --bootstrap-leader-lamports "$stake"
         --bootstrap-leader-stake-lamports "$stake"
-        --lamports-per-signature 1
```

Comment:
this is the only line that needs to be removed

Reply:
Got it.. I'll update the patch

Reply:
Done